src/handlers/gestures/simpleGestures.ts

```diff
@@ -0,0 +1,429 @@
+    if (Array.isArray(offset)) {
+      this.config.failOffsetYStart = offset[0];
+      this.config.failOffsetYEnd = offset[1];
+    } else if (offset < 0) {
+      this.config.failOffsetYStart = offset;
+    } else {
+      this.config.failOffsetYEnd = offset;
+    }
+    return this;
+  }
+
+  setFailOffsetX(offset: number | number[]) {
+    if (Array.isArray(offset)) {
+      this.config.failOffsetXStart = offset[0];
+      this.config.failOffsetXEnd = offset[1];
+    } else if (offset < 0) {
+      this.config.failOffsetXStart = offset;
+    } else {
+      this.config.failOffsetXEnd = offset;
+    }
+    return this;
+  }
+
+  setMinDistance(distance: number) {
+    this.config.minDist = distance;
+    return this;
+  }
+
+  setAverageTouches(value: boolean) {
+    this.config.avgTouches = value;
+    return this;
+  }
+
+  setEnableTrackpadTwoFingerGesture(value: boolean) {
+    this.config.enableTrackpadTwoFingerGesture = value;
+    return this;
+  }
+}
+
+export class Pinch extends SimpleGesture {
+  constructor() {
+    super();
+
+    this.handlerName = 'PinchGestureHandler';
+  }
+
+  setOnUpdate(callback: (event: PinchGestureHandlerEvent) => void) {
+    this.handlers.onUpdate = callback;
+    return this;
+  }
+}
+
+export class Rotation extends SimpleGesture {
+  constructor() {
+    super();
+
+    this.handlerName = 'RotationGestureHandler';
+  }
+
+  setOnUpdate(callback: (event: RotationGestureHandlerEventPayload) => void) {
+    this.handlers.onUpdate = callback;
+    return this;
+  }
+}
+
+type LongPressGestureConfig = CommonGestureConfig & {
+  minDurationMs: number;
+  maxDist: number;
+};
+
+export class LongPress extends SimpleGesture {
+  public config: Partial<LongPressGestureConfig> = {};
+
+  constructor() {
+    super();
+
+    this.handlerName = 'LongPressGestureHandler';
+  }
+
+  setMinDuration(duration: number) {
+    this.config.minDurationMs = duration;
+    return this;
+  }
+
+  setMaxDistance(distance: number) {
+    this.config.maxDist = distance;
+    return this;
+  }
+}
+
+type FlingGestureConfig = CommonGestureConfig & {
+  numberOfPointers: number;
+  direction: Directions;
+};
+
+export class Fling extends SimpleGesture {
+  public config: Partial<FlingGestureConfig> = {};
+
+  constructor() {
+    super();
+
+    this.handlerName = 'FlingGestureHandler';
+  }
+
+  setNumberOfPointers(pointers: number) {
+    this.config.numberOfPointers = pointers;
+    return this;
+  }
+
+  setDirection(direction: Directions) {
+    this.config.direction = direction;
+    return this;
+  }
+}
+
+type ForceTouchGestureConfig = CommonGestureConfig & {
+  minForce: number;
+  maxForce: number;
+  feedbackOnActivation: boolean;
+};
+
+export class ForceTouch extends SimpleGesture {
+  public config: Partial<ForceTouchGestureConfig> = {};
+
+  constructor() {
+    super();
+
+    this.handlerName = 'ForceTouchGestureHandler';
+  }
+
+  setOnUpdate(callback: (event: ForceTouchGestureHandlerEventPayload) => void) {
```

Comment:
You can use `override` keyword in TS to signify that.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-3.html#override-and-the---noimplicitoverride-flag

Reply:
This method is defined only in continuous gestures, not in the parent class.